client/src/components/RandomizedValue.tsx

```diff
@@ -9,18 +9,17 @@ interface Props{
   rerollFunction: () => void
 }
 
+// Naming is odd, it sounds like a value, not like a component.
```

Comment:
can you suggest a better name? i thought naming components was supposed to be generic and "Randomized Value" was as generic as i could think of

Reply:
It's a component, so it needs to be named as something that you give you the feeling that it will be printed.
If it has no Dom associated, it should not be a comp.
`RandomizedValueChip`? 
Something like that that fit your project